src/content/admin/cloud-credentials/aws.mdx

```diff
@@ -37,6 +28,15 @@ Okteto displays the OIDC endpoint of your cluster in the Admin -> General view o
      />
    </p>
 
+```bash
```

Comment:
I would present it this way: first showing where to obtain the data and then how to use it